python/tests/table/test_metadata.py

```diff
@@ -76,12 +76,12 @@ def example_table_metadata_v1() -> Dict[str, Any]:
 
 def test_from_dict_v1(example_table_metadata_v1: Dict[str, Any]) -> None:
     """Test initialization of a TableMetadata instance from a dictionary"""
-    TableMetadataUtil.parse_obj(example_table_metadata_v1)
+    TableMetadataUtil.parse_raw(json.dumps(example_table_metadata_v1))
```

Comment:
I think for testing we're okay with leaving as it was.

Reply:
So, we are going to have a `parse_obj` method that works like before. Is that correct?

Reply:
Yes, for the tests I think `parse_obj` is fine

Reply:
Ok, but still I have to add tests for the new method. So, it's like duplicating the test but call another method